### DISCUSSION TITLE
MultipleFilesCreated

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#4958] [Help]
Im using magickwand api to resize the image

Files are being created in the image of imageId
while checking found those files are created after using magickwand api and those files not having any data and those files are not deleted

is this expected ?
in imagemagick do we create any such file apart from the files in the temporary-path ?

**Comment by snibgo** [comment]
Sorry, I don't understand the problem.

What files are created? Please show your MagickWand code, and all the input and output files.

**Reply by programmerarunkumar** [reply]
I'm using MagickWand Method to resize the Image

NewMagickWand
MagickReadImage
MagickResizeImage
MagickWriteImages
DestroyMagickWand

input : imageId, original image path, resized image path

some unknown files are created in the name of imageId 
after calling the above mentioned magickwand method that unknown file is created

**Comment by snibgo** [comment]
If you don't show your code or input files or created files, I can't help you.
